README.md

```diff
@@ -149,7 +149,7 @@ SchemaPlus::ForeignKeys is tested on:
 
 ## History
 
-* 1.0.0 - Compatibility with ActiveRecord 5.0. Removed support for 4.x
+* 1.0.0 - Compatibility with ActiveRecord 5.0.
```

Comment:
Oh, and why make this a major version bump?  Any non-backwards-compatible changes?

If everything still works the same, I'd argue this should be 0.1.7 or maybe 0.2.0 if we consider AR5 support to be a new API feature.